pygluu/kubernetes/templates/helm/gluu/charts/fido2/templates/_helpers.tpl

```diff
@@ -67,6 +67,18 @@ Create user custom defined secret envs
 {{- end }}
 {{- end }}
 
+{{/*
+Create GLUU_JAVA_OPTIONS ENV for passing custom work and detailed logs
+*/}}
+{{- define "fido2.customJavaOptions"}}
+{{ $custom := "" }}
+{{- if .Values.global.fido2.gluuCustomJavaOptions }}
```

Comment:
This comment applies to all services . 


You need to set the Java settings directly in the Java options env pre calculated from the resource limits not through a variable in the values.yaml as that will clash with the resource definitions .